wafer/users/templates/wafer.users/registration_confirm_mail.txt

```diff
@@ -0,0 +1,12 @@
+{% load i18n %}
+{% blocktrans %}
+Thank you for registering for {{ conference_name }}.
```

Comment:
Why conference_name, rather than WAFER_CONFERENCE_NAME as in all the other templates?